content/blog/review/books-simplified-embedded-rust/index.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Add a cover image, named `featured.png` on the same level as the `index.md`.
The image size could be: 1200x600px

Reply:
Added webp